lib/lang/latin/latin-language-dataset.js

```diff
@@ -279,7 +279,7 @@ export default class LatinLanguageDataset extends LanguageDataset {
       const item = data[i]
       let lemma = item[0]
       // let principalParts = item[1].split(/_/)
-      let principalParts = item[1]
+      // let principalParts = item[1]
```

Comment:
(And then in the view we would need to set the title of the view using the Lemma object)

Reply:
Oh, yes. I didn'y understand you correctly before - because it was commented before I started to work on the task, so I didn't think that we need it. I will make these changes.